docs/en/resources/sources/alloydb-admin.md

```diff
@@ -36,7 +35,9 @@ sources:
 ```
 
 ## Reference
+
 | **field**      | **type** | **required** | **description**                                                                                                                                |
-|----------------|:--------:|:------------:|------------------------------------------------------------------------------------------------------------------------------------------------|
+| -------------- | :------: | :----------: | ---------------------------------------------------------------------------------------------------------------------------------------------- |
 | kind           |  string  |     true     | Must be "alloydb-admin".                                                                                                                       |
-| useClientOAuth | boolean  |    false     | If true, the source will use client-side OAuth for authorization. Otherwise, it will use Application Default Credentials. Defaults to `false`. |
\ No newline at end of file
+| defaultProject |  string  |     false    | The environment variable containing the Google Cloud project ID to use for all AlloyDB operations.                                             |
```

Comment:
The prebuilt tools read the env var but in general this could be hard coded for custom tools